vj4/handler/base.py

```diff
@@ -51,6 +51,11 @@ class HandlerBase(setting.SettingMixin):
           domain.get(self.domain_id),
           domain.get_user(self.domain_id, uid),
           blacklist.get(self.remote_ip))
+      sentry = app.Application().sentry_middleware
+      if sentry:
+        sentry.client.user_context({
+          '_id': uid
```

Comment:
single line, or put comma at the end. i.e. either

`sentry.client.user_context({'_id': uid})`

or

```
sentry.client.user_context({
  '_id': uid,
})
```